doc/languages-frameworks/python.section.md

```diff
@@ -1482,29 +1482,29 @@ If you need to change a package's attribute(s) from `configuration.nix` you coul
   nixpkgs.config.packageOverrides = super: {
     python = super.python.override {
       packageOverrides = python-self: python-super: {
-        zerobin = python-super.zerobin.overrideAttrs (oldAttrs: {
-          src = super.fetchgit {
-            url = "https://github.com/sametmax/0bin";
-            rev = "a344dbb18fe7a855d0742b9a1cede7ce423b34ec";
-            sha256 = "16d769kmnrpbdr0ph0whyf4yff5df6zi4kmwx7sz1d3r6c8p6xji";
+        scipy = python-super.scipy.overrideAttrs (oldAttrs: {
+          src = super.fetchPipy {
+            pname = "scipy";
+            version = "1.5.0";
+            sha256 = "4ff72877d19b295ee7f7727615ea8238f2d59159df0bdd98f91754be4a2767f0";
           };
         });
       };
     };
   };
 ```
 
-`pythonPackages.zerobin` is now globally overridden. All packages and also the
-`zerobin` NixOS service use the new definition. Note that `python-super` refers
```

Comment:
It might be worth preserving the mention that NixOS services will also inherit overrides, if there's a Python package that's also used in a NixOS service and we can use that instead of scipy.

Looking at `git grep -E "ExecStart.*python.?Packages\." -- nixos/modules/` .. twisted can be a candidate.

```
nixos/modules/services/monitoring/graphite.nix:          ExecStart = "${pkgs.python3Packages.twisted}/bin/twistd ${carbonOpts name}";
```

Both graphite and twisted will probably be around for a while.

Reply:
@ento I've replaced `scipy` by `twisted` as you suggested.